cpp/src/parquet/properties.h

```diff
@@ -351,40 +371,57 @@ class PARQUET_EXPORT WriterProperties {
       return this->compression_level(path->ToDotString(), compression_level);
     }
 
+    /// Define the file encryption properties.
+    /// Default NULL.
     Builder* encryption(
         std::shared_ptr<FileEncryptionProperties> file_encryption_properties) {
       file_encryption_properties_ = std::move(file_encryption_properties);
       return this;
     }
 
+    /// Enable statistics in general.
+    /// Default enabled.
     Builder* enable_statistics() {
       default_column_properties_.set_statistics_enabled(true);
       return this;
     }
 
+    /// Disable statistics in general.
+    /// Default enabled.
     Builder* disable_statistics() {
       default_column_properties_.set_statistics_enabled(false);
       return this;
     }
 
+    /// Enable statistics for the column specified by `path`.
+    /// Default enabled.
     Builder* enable_statistics(const std::string& path) {
       statistics_enabled_[path] = true;
       return this;
     }
 
+    /// Enable statistics for the column specified by `path`.
+    /// Default enabled.
     Builder* enable_statistics(const std::shared_ptr<schema::ColumnPath>& path) {
       return this->enable_statistics(path->ToDotString());
     }
 
+    /// Disable statistics for the column specified by `path`.
+    /// Default enabled.
     Builder* disable_statistics(const std::string& path) {
       statistics_enabled_[path] = false;
       return this;
     }
 
+    /// Disable statistics for the column specified by `path`.
+    /// Default enabled.
     Builder* disable_statistics(const std::shared_ptr<schema::ColumnPath>& path) {
       return this->disable_statistics(path->ToDotString());
     }
 
+    /// \brief This function builds the WriterProperties according to the parameters of
+    /// builder.
```

Comment:
nit: write the docstring imperatively, so 'Build the WriterProperties with the builder parameters' or something like that.